aws/ecs-service/files/container-instance-assume-policy.json

```
@@ -0,0 +1,13 @@
+{
+    "Version": "2008-10-17",
```

```suggestion
    "Version": "2012-10-17",
```

Can we use the [newest version](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_elements_version.html) here?

---

It'll work, as we're not using any of the new policy syntax, but AWS recommends against using it; 

> `2008-10-17`. This was an earlier version of the policy language. You might see this version on older existing policies. Do not use this version for any new policies or when you update any existing policies.

This is something they haven't refined on the autogenerated policies from the console.